gradle/java.gradle

```diff
@@ -33,16 +34,7 @@ ext.powerMockDependencies = [
     "org.powermock:powermock-reflect:$powerMockVersion"
 ]
 
-ext.apachedsDependencies = [
```

Comment:
Is there any reason for removing this? It seems that its removal has caused quite a bit of duplication.